dbt/task/compiler/linker.py

```diff
@@ -0,0 +1,93 @@
+import sqlparse
```

Comment:
That sounds like speculative design.  We can always break it off into its own file later, if we decide we need to.